tools/gsc-mannwhitney_de/mannwhitney_de.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="mannwhitney_de" name="Perform a differential analysis" version="0.1.0">
+    <description>between the two groups of a signature score using a Mann-Whitney test</description>
+    <requirements>
+        <requirement type="package" version="1.3.2=r3.3.2_0">r-optparse</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Tool exception" />
+    </stdio>
+    <command detect_errors="exit_code"><![CDATA[ 
+        Rscript $__tool_directory__/MannWhitney_DE.R 
+            --input $input 
```

Comment:
`'$input'` systematically quote input variable (and output variable because it does not hurt) is a best practice, to avoid passing malignant code to the tool

Reply:
Ok !